src/Components/AllIngredientsHandler.cs

```diff
@@ -42,173 +36,322 @@ public AllIngredientsHandler(IBackend backend, IUser user)
+            SolidColorBrush boxForeground = (SolidColorBrush)App.Current.Resources["SecondaryBrushB"];
+            int boxFont = 24;
+
+            // Create the search box
             TextBox searchBox = new TextBox
             {
-                Height = 30,
-                Foreground = Brushes.Black,
-                Background = Brushes.White,
-                Margin = new Thickness(0, 10, 0, 10),
-                Text = "Search ingredients..."
+                Height = 40,
+                Margin = new Thickness(25, 10, 30, 10),
+                Foreground = boxForeground,
+                Background = boxBackground,
+                HorizontalAlignment = HorizontalAlignment.Stretch,
+                VerticalAlignment = VerticalAlignment.Center,
+                VerticalContentAlignment = VerticalAlignment.Center,
+                Text = "Search ingredients...",
+                FontSize = boxFont
             };
 
-            // Clear placeholder on focus
+            // Clear placeholder text when the box is focused
             searchBox.GotFocus += (s, e) =>
             {
-                if (searchBox.Text == "Search ingredients...") searchBox.Text = string.Empty;
+                if (searchBox.Text == "Search ingredients...")
+                {
+                    searchBox.Text = string.Empty;
+                }
             };
-            searchBox.LostFocus += (s, e) =>
+
+            // Restore placeholder text if the box is empty when focus is lost
+            searchBox.LostFocus += async (s, e) =>
             {
-                if (string.IsNullOrEmpty(searchBox.Text)) searchBox.Text = "Search ingredients...";
+                if (string.IsNullOrEmpty(searchBox.Text))
+                {
+                    searchBox.Text = "Search ingredients...";
+                    await DisplayIngredientsAsync(parentPanel);
+                }
             };
 
             return searchBox;
         }
 
+
         /// <summary>
         /// Creates a row with ingredient details and an add button, wrapped in a border.
         /// </summary>
         /// <param name="ingredient">The ingredient to display in the row.</param>
         /// <returns>A styled Border containing the ingredient row.</returns>
         private Border CreateIngredientRow(Ingredient ingredient)
         {
-            DockPanel ingredientRow = new DockPanel { Margin = new Thickness(0, 5, 0, 5) };
-            TextBlock ingredientText = new TextBlock
+            // Define resource-based colors and font sizes
+            SolidColorBrush boxTextCol = (SolidColorBrush)App.Current.Resources["SecondaryBrushB"];
+            SolidColorBrush boxBorderCol = (SolidColorBrush)App.Current.Resources["SecondaryBrushB"];
+            int boxTextFont = 28;
+            int boxButtonFont = 32;
+
+            // Create ingredient row 
+            DockPanel ingredientRow = CreateIngredientRowPanel();
```

Comment:
Instead of putting the Ingredient name, type and icons to the left of each other, could you place it on the bottom instead so that we can actually see the full ingredient name, type, unit and amount? Similar to the mobile frontend:
![image](https://github.com/user-attachments/assets/bfcea32f-9ee6-4c00-9495-f643c0132687)